editor/layouts/layout-tools.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: 'Tools'
+sidebarTitle: 'Tools'
+description: 'There are several Layout tools available in Rive to build your responsive UI or content'
+---
+
+import { YouTube } from '/snippets/youtube.mdx'
+
+## Getting Started with Layouts
+
+There are a number of ways to start adding layouts to your designs.
+
+- Layout Tools in the Arrangement Tools menu
+- Wrap in Layout
+- Add Child Layout
+- Dragging and Dropping Layouts
+
+---
+
+### Layout Tools in the Arrangement Tools Menu
+
+A selection of layout options are available in the Arrangement Tools menu. These tools allow you to draw a Layout in 2 ways:
+
+1. When dragged directly onto the Artboard, it will create an Absolute Layout positioned relative to the Top Left of the Artboard.
+2. When dragged on top of another Layout, it will create a Relative child Layout inside the Layout it was drawn on top of.
+
+![Image](https://ucarecdn.com/53f42ad0-4045-41c4-bcfe-90a40ad0dc1d/)
+
+- **Layout:** A single Layout container. Select the tool and drag on an artboard to create the Layout. As mentioned above, drag directly onto the Artboard to create an Absolute Layout or on top of an existing Layout to create a Relative child. In addition, any objects positioned within the bounds of the newly created Layout will automatically themselves be wrapped in Layouts and absolutely positioned inside the new Layout. 
+
+After creating an Absolute Layout, you can drag it to reposition it, or resize it using the resize handles.
+
+<Tip>Absolute Layouts can be dragged around like other Rive objects. By default, if you drag a Layout (or any other object) inside the bounds of another Layout, it will display an indicator showing you that the dragged object will become a child of that Layout if dropped. Hold `command` / `control` while dragging the object to prevent existing items from being moved into the Layout.</Tip>
+
+  <video width="700" controls>
+    <source src="https://ucarecdn.com/994d9855-4783-478f-a808-e5f8e57511af/" type="video/mp4"></source>
```

Comment:
Same thing, for video we've been using YouTube for hosting, but this looks really interesting